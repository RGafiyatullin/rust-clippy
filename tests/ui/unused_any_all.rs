#![warn(clippy::unused_any_all)]

fn main() {
    (0..1).any(|_| false);
    Iterator::any(&mut (0..1), |_| false);
    let _ = (0..1).any(|_| false);
    let _ = Iterator::any(&mut (0..1), |_| false);
}
