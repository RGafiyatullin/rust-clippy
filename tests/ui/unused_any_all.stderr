error: unused result of `Iterator::any`
  --> tests/ui/unused_any_all.rs:4:5
   |
LL |     (0..1).any(|_| false);
   |     ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D clippy::unused-any-all` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::unused_any_all)]`

error: unused result of `Iterator::any`
  --> tests/ui/unused_any_all.rs:5:5
   |
LL |     Iterator::any(&mut (0..1), |_| false);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

